### PR TITLE
Fixed padding issue

### LINKED ISSUE #8196: [BUG] doc page need padding between horizontal line and hover line
### Description

When the user is in dark mode and hovers over the card below the horizontal line, it needs some padding.

**Steps to Reproduce:**

1. Enable dark mode in the application.
3. Navigate to the doc page.
Hover over the cards that are exactly below the horizontal line.

### Screenshots

bug

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by wx-it**
## Fixed Issue of padding between horizontal line and hover line 

fixes #8196

### Changes proposed
- Added a margin top to the cards, and it fixed the issue.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8236"></a>

### CODE REVIEW

File: pages/docs/index.js [hunk 1]
```diff
@@ -126,10 +126,11 @@ export default function DocsIndex() {
                   </div>
                   <div className="mt-12 grid gap-16 lg:grid-cols-3 lg:gap-x-5 lg:gap-y-12">
                     {section.pages.map((page) => (
+                      //added margin top to fix the issue
```
Comment (eddiejaoude) [inline pages/docs/index.js:129]: This comment is not needed, otherwise we would have so many, one for each class

File: pages/docs/index.js [hunk 2]
```diff
@@ -126,10 +126,11 @@ export default function DocsIndex() {
...
                       <Link
                         href={page.path}
                         key={page.name}
-                        className="border border-transparent hover:border hover:border-orange-600 transition-all duration-250 ease-linear rounded px-6 py-2 block"
+                        className="border border-transparent hover:border hover:border-orange-600 transition-all duration-250 ease-linear rounded px-6 py-2 mt-4 block"
```
Comment (eddiejaoude) [inline pages/docs/index.js:133]: I don't think it need to be added to every link item? The ones on the 2nd row will now have a bigger gap at the top and that is not required. Suggest adding to the wrapping item

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution, but I think the margin should be move - I have left inline comments

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
The indentation doesn't look correct?

Please also add a screenshot of before and after

**Comment by SaraJaoude** [comment]
As there has been no activity for two weeks I will be closing this PR.

**Pull request closed without merging**